srcpkgs/mesa/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'mesa'
 pkgname=mesa
-version=22.3.5
-revision=2
+version=23.0.2
+revision=1
 build_style=meson
 configure_args="-Dglvnd=true -Dshared-glapi=enabled -Dgbm=enabled -Degl=enabled
  -Dosmesa=true -Dgles1=enabled -Dgles2=enabled -Dglx=dri -Ddri3=enabled
  -Dlmsensors=enabled -Dplatforms=x11$(vopt_if wayland ,wayland)
- -Dllvm=enabled -Db_lto=false -Dcpp_std=gnu++14"
+ -Dllvm=enabled -Db_lto=false"
```

Comment:
I built and tested this locally without `-Db_lto=false`, didn't notice any issues whatsoever:
```suggestion
 -Dllvm=enabled"
```

Reply:
https://github.com/void-linux/void-packages/pull/41084#issuecomment-1430381994

Reply:
Makes sense, I'd link to https://gitlab.freedesktop.org/mesa/mesa/-/issues/6911 in a comment above `configure_args` to clarify the reason this is needed.